src/MissionManager/QGCMapPolygonVisuals.qml

```diff
@@ -576,8 +577,15 @@ Item {
             z:                  QGroundControl.zOrderMapItems + 1   // Over item indicators
 
             onClicked: {
-                if (mouse.button === Qt.LeftButton && _root.interactive) {
-                    mapPolygon.appendVertex(mapControl.toCoordinate(Qt.point(mouse.x, mouse.y), false /* clipToViewPort */))
+                if(_utmspEnabled){
+                    if (mouse.button === Qt.LeftButton) {
+                        mapPolygon.appendVertex(mapControl.toCoordinate(Qt.point(mouse.x, mouse.y), false /* clipToViewPort */))
```

Comment:
Please, use this instead of comment inside of the arguments
bool clipToViewPort = false
mapPolygon.appendVertex(..., clipToViewPort);